zap/src/main/java/org/parosproxy/paros/extension/option/ExtensionOption.java

```diff
@@ -161,4 +166,29 @@ public String getAuthor() {
     public boolean supportsDb(String type) {
         return true;
     }
+
+    /**
+     * Gets an options button, accepting a tooltip and options panel name (to be displayed when
+     * opening the options dialog).
+     *
+     * <p>Should be called/used only when in view mode.
+     *
+     * @param tooltip the text to be displayed in the button's tooltip.
+     * @param optionsPanelName the name of the panel to be displayed when the Options dialog panel
+     *     is opened.
+     * @return the options button, never {@code null}.
+     * @since TODO Add version
```

Comment:
Can be replaced with the actual version (if we happen to use other version we have to replace anyway, if not less work).